### ISSUE TITLE
Cannot stop RNode due to error:nonce too low

### ISSUE THREAD

**Issue by Somstradamus** [CPChain/chain#110]
Hi there, 
This is CPChain team. Note that this is for bug report. If you are requesting a feature, please select *Feature Request* template. 

**Where does the bug occur?**
RNode

**What is your wallet address and your balance?**
Too private

**Describe the bug**
I'm trying to stop my RNode with this command:

C:\Users\Admin\Desktop>cpchain-windows-4.0-amd64.exe campaign stop --keystore E:\Cpchain\keystore\UTC--2019-10-09T16-13-51.955Z--e4xxxxxxxxxxxxxxxxxxxxx7a81fe2
Input password:
Password: ***********

but the outcome is this error:

INFO[07-25|12:25:44.968] Stop Mining...
INFO[07-25|12:25:44.994] Stop Mining Success
INFO[07-25|12:25:44.994] Quit Rnode...
INFO[07-25|12:25:45.002] error:nonce too low

**To Reproduce**
Type in a command shell under Windows 10 the following command while the RNode is running (locally): 
C:\Users\Admin\Desktop>cpchain-windows-4.0-amd64.exe campaign stop --keystore E:\Cpchain\keystore\UTC--2019-10-09T16-13-51.955Z--e4f8bec7cd890axxxxxxxxxxxxxxxxxxxxxxxxx

**Expected behavior**
The error above will pop up.

** Configuration (please complete the following information):**
 - OS: Windows 10
 - cpchain version cpchain-windows-4.0-amd64

**Additional context**
I have typed this command to run the RNode in order to be able to stop it in parallel:

C:\Users\Admin\Desktop>cpchain-windows-4.0-amd64.exe run --rpcaddr 127.0.0.1:8501 --rpcapi personal,eth,cpc,admission,net,web3,db,txpool,miner

**Comment by cpchainbot** [comment]
Sorry……

We can not reproduce this error under Windows 10. If this error occurs again, please create a new issue.

**Comment by cpchainbot** [comment]
Or you can submit a suggestion on our latest CPC-Wallet and submit your email. We will help you to quit RNode and take your CPC back from the RNode contract.
